### PR TITLE
Create "Open source subsidy program" proposal

### PR DESCRIPTION
**Pull request by coni2k**

### CODE REVIEW

**Comment by coni2k** [comment]
Dear @epicfaace,
This proposal is also ready for review.
Thank you!

**Comment by epicfaace** [comment]
Thanks, taking a look!

--
Ashwin Ramaswami

File: proposals/open-source-subsidy-program.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+2. **How is this proposal innovative -- what distinguishes it from other related work?** No well-known study provides a detailed framework for implementing a subsidy program and tackling its practical challenges.
+3. **Who is your doer -- who will execute the proposed work?** It may depend on the extensiveness of the study. I'm willing to take responsibility for the execution. However, the proposal contains multiple areas, and collaborating with professional research organizations can be ideal.
+4. **How might this work be sustained long-term after an initial seed grant?** Providing the framework for this subsidy program should be a one-time study. The related government agencies should carry out the program's implementation if any find it feasible.
+
+### Resources needed
+
+A grant to allow us to carry out this study. It's possible to split the research into smaller focus areas; hence the amount could differ based on the extensiveness of the study. Providing contacts from existing research organizations would also be helpful.
```
Comment (epicfaace) [inline proposals/open-source-subsidy-program.md:80]: What exactly would this study fund? Such as researchers, or the actual money to be disbursed through subsidies, etc.?

**Review by epicfaace — COMMENTED** [review]
lgtm -- thanks! @deepesch any thoughts before we merge?

File: proposals/open-source-subsidy-program.md (hunk 1 above)
Comment (coni2k) [reply proposals/open-source-subsidy-program.md:80]: I thought to use the fund only for the researchers and the study itself.

On the other hand, we could also consider running a small pilot program to test the idea in practice once the study is over.

At the end of the study, we should have a framework/guidelines for running such a subsidy program. Once/if the government implements the program, they should allocate the resources and distribute the subsidy to the companies.

**Pull request merged**